openrtb_ext/imp_yandex.go

```diff
@@ -0,0 +1,16 @@
+package openrtb_ext
+
+type ExtImpYandex struct {
+	/*
+		Possible formats
+			- `R-I-123456-2`
+			- `R-123456-1`
+			- `123456-789`
+	*/
+	PlacementID string `json:"placement_id"`
```

Comment:
I've extended the ext with `placementID` to so that the params would match with our PBJS adapter